daml-assistant/src/DAML/Assistant/Install.hs

```diff
@@ -57,6 +59,7 @@ data InstallEnv = InstallEnv
     , targetVersionM :: Maybe SdkVersion
     , damlPath :: DamlPath
     , projectPathM :: Maybe ProjectPath
+    , out :: Handle
```

Comment:
Why not `output :: (String -> IO ())`? Restricting it to a handle is still pretty restrictive.

Reply:
:+1: